whylog/assistant/regex_assistant/regex_object.py

```diff
@@ -8,69 +10,80 @@ class RegexObject(object):
     Supports interactive updates like adding/removing groups to regex, replacing regex
     Keeps own data integrity - especially consistency between regex and groups (group_spans).
     Verifies updates.
-    Not responsible for guessing where are groups in regex
+
+    :param line_text: line text (raw string)
```

Comment:
This discription inform about nothing. You can change it to 'just raw string of the line'.